src/core/js/models/itemsModel.js

```diff
@@ -43,6 +44,61 @@ define([
             if (this.getCompletionStatus()) {
                 this.setCompletionStatus();
             }
+        },
+
+        resetActiveItems: function(trigger) {
+            var _trigger = (trigger === undefined) ? true : trigger;
+            var items = this.get('_items');
+
+            _.each(items, function(item) {
+                item._isActive = false;
+            });
+
+            if (_trigger) this.trigger('change:_items:_isActive', this, items);
+        },
+
+        getActiveItems: function() {
+            return _.filter(this.get('_items'), function(item) {
+                return item._isActive;
+            });
+        },
+        
+        getActiveItemsCount: function() {
+            return this.getActiveItems().length;
+        },
+        
+        getActiveItemsIndexes: function() {
+            var indexes = [];
+            var items = this.get('_items');
+            
+            for (var i = 0, l = items.length; i < l; i++)
+                if (items[i]._isActive) indexes.push(i); 
+
+            return indexes;
+        },
+
+        setItemAtIndexAsActive: function(index, trigger) {
+            var _trigger = (trigger === undefined) ? true : trigger;
+            var items = this.get('_items');
+            var item = items[index];
+            
+            if (item === undefined) return false;
+
+            item._isActive = true;
+            if (_trigger) this.trigger('change:_items:_isActive', this, items);
+            return item;
+        },
+        
+        setItemAtIndexAsInactive: function(index, trigger) {
+            var _trigger = (trigger === undefined) ? true : trigger;
+            var items = this.get('_items');
+            var item = items[index];
+            
+            if (item === undefined) return false;
+
+            item._isActive = false;
+            if (_trigger) this.trigger('change:_items:_isActive', this, items);
```

Comment:
```if (trigger)``` will work as undefined equals to false in an if statement, otherwise known as a "falsy" http://adripofjavascript.com/blog/drips/truthy-and-falsy-values-in-javascript.html;